public/app/features/admin/UserListPage.tsx

```diff
@@ -10,31 +12,84 @@ import { AccessControlAction } from '../../types';
 import { UsersListPageContent } from '../users/UsersListPage';
 
 import { UserListAdminPageContent } from './UserListAdminPage';
+import { UserListPublicDashboardPage } from './UserListPublicDashboardPage/UserListPublicDashboardPage';
+
+enum TabView {
+  ADMIN = 'admin',
+  ORG = 'org',
+  PUBLIC_DASHBOARDS = 'public-dashboards',
+}
+
+const selectors = e2eSelectors.pages.UserListPage;
+
+const PublicDashboardsTab = ({ view, setView }: { view: TabView | null; setView: (v: TabView | null) => void }) => (
```

Comment:
Personal preference: I like to have the props out of the component function. It is easier to read.